Source/OCcore/RawDump/rawdump.cpp

```diff
@@ -0,0 +1,29 @@
+//
+// Created by supragya on 13/4/18.
+//
+
+#include <fstream>
+#include <iostream>
+#include "rawdump.h"
+#include "Log/Logger.h"
+
+bool OcRawDump::dump(std::string fileName, unsigned char* data, long length){
+    std::ofstream os;
+    os.open(fileName, std::ios::binary);
+    if (!os.is_open())
+    {
+        std::string err = "OcRawDump: File couldn't be opened: ";
+        err.append(fileName);
+        OC_LOG_ERROR(err);
+        return false;
+    }
+    std::string begin = "OcRawDump: Begin dumping in: ";
+    begin.append(fileName);
```

Comment:
fileName can also be added by using +, e.g. ".... be opened: " + fileName;

Reply:
corrected in #38